tests/system/python/test_e2e_coap_OCS.py

```diff
@@ -0,0 +1,105 @@
+# -*- coding: utf-8 -*-
```

Comment:
name it `test_e2e_OCS.py` we should be able to pass south plugin name from supported list in longer term